application/worker.py

```diff
@@ -449,7 +457,17 @@ def run_task(t, args, aggregate_model=False):
                 t.repo_dir = gherkin_repo_dir
 
         try:
+            from datetime import datetime
+            start_time = datetime.now()
             task(d, *args)
+            end_time = datetime.now()
+            if t.__name__ != "general_info_task":
```

Comment:
Instead of hardcoding this I would rather prefer you try-catch the attribute lookup or use `hasattr`.